torch/csrc/jit/api/object.h

```diff
@@ -34,6 +35,12 @@ struct TORCH_API Object {
     return _ivalue()->type();
   }
 
+  struct ObjectProperty {
+    std::string name;
+    Method getter_func;
+    c10::optional<Method> setter_func;
+  };
```

Comment:
This no longer needs to be called `ObjectProperty` since it is inside `Object`. Also, please add documentation for what this struct is/does.
```suggestion
  struct Property {
    std::string name;
    Method getter_func;
    c10::optional<Method> setter_func;
  };
```